src/components/NavBar/index.tsx

```diff
@@ -4,7 +4,7 @@ import Web3Status from 'components/Web3Status'
 import { useIsPoolsPage } from 'hooks/useIsPoolsPage'
 import { Box } from 'nft/components/Box'
 import { Row } from 'nft/components/Flex'
-import { UniIcon } from 'nft/components/icons'
+import { HorsIcon } from 'nft/components/icons'
```

Comment:
The `s` is from swap, not from the `horse`.  😀 
```suggestion
import { HorIcon } from 'nft/components/icons'
```